projects/vdk-control-cli/src/vdk/internal/control/job/sample_job/config.ini

```diff
@@ -17,6 +17,11 @@ team =
 ; the cron job waits until the previous execution has finished.
 schedule_cron = 11 23 5 8 1
 
+;; Specifies the python version to be used in the job deployment.
```

Comment:
How to find available options ?

Reply:
How to find out the default version as well (in order to decide if I want to change it).

Reply:
Users can see the python version that a job is deployed with if they use `vdk deploy --show ...`, or through the API. This is not implemented in the UI, though (at least not at this point in time).